BotRoyaleUnity/Assets/Scripts/Networking/SocketConnectionHandler.cs

```diff
@@ -92,7 +92,9 @@ private void HandleGameMessageReceived(SocketIOEvent socketEvent)
         }
         else
         {
-            Debug.LogWarning("A game-message with action " + actionName + " was received, but there are no listeners for this action");
+            if(actionName != "currentParts"){
```

Comment:
Is this because the Unity client will never need to do anything when currentParts actions happen?

Reply:
Yeah, and it was creating a warning in the unity editor every time a message was sent.